UI/ad-grp.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
Indent the contents of the `head` tag for all files
Something like this
```
<head>
    <title>Epic Email</title>
    <link rel="stylesheet" type="text/css" href="css/style.css">
    <meta charset="utf-8"> 
    <META name="keywords" lang="en-us" content="emails">
    <meta name="viewport" content="width=device-width, initial-scale=1">
</head>
```

Reply:
Resolved